From 8a42d91d984aca89ac4b302a2aeeb37674c1c0da Mon Sep 17 00:00:00 2001
From: Oliver Brown <oliver.brown@freescale.com>
Date: Tue, 10 Sep 2013 01:04:49 -0500
Subject: [PATCH 0168/1074] ENGR00278920 [ARM] - dts update MIPI CSI2 node on
 SabreAuto

Correct the SabreAuto DTS.

Signed-off-by: Oliver Brown <oliver.brown@freescale.com>
[Original patch taken from git://git.freescale.com/imx/linux-2.6-imx.git]
Signed-off-by: Biyao Zhai <biyao.zhai@windriver.com>
---
 arch/arm/boot/dts/imx6qdl-sabreauto.dtsi |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
index d9504e9..b34c6a2 100644
--- a/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
+++ b/arch/arm/boot/dts/imx6qdl-sabreauto.dtsi
@@ -366,7 +366,7 @@
 	fsl,cpu_pdnscr_iso = <0x1>;
 };
  
-&mipi {
+&mipi_csi {
 	status = "okay";
 	ipu_id = <0>;
 	csi_id = <1>;
-- 
1.7.5.4

